src/map/clif.c

```diff
@@ -9851,21 +9851,37 @@ static int clif_guess_PacketVer(int fd, int get_previous, int *error)
 		err = 0;\
 		if( error )\
 			*error = 0;\
+		if( diskey ) ShowWarning("Client use no encryption while server do, please fix me\n"); \
```

Comment:
I think we should not add this after all. It helps us to see if we used a wrong client during development, but for a production server that one is a no-go. If a server-owner wants players to use packet obfuscation and players try to connect with a custom diffed client to their server, the server gets flooded by those messages. So either we do it like hercules, that we add an option for supporting obfuscated and plain connections at the same time(which must be possible to be disabled by a setting) or just leave it out in general.